tests/e2e/e2e_test.go

```diff
@@ -725,7 +726,7 @@ var _ = Describe("odoe2e", func() {
 		})
 
 		It("should delete the deployed image-specific component", func() {
-			runCmdShouldPass("odo delete testversioncmp")
+			runCmdShouldPass("odo delete testversioncmp -f")
```

Comment:
was this working before? what I mean is without the `--force` flag this command would have just blocked on the user confirmation. so even then the test was passing?

Reply:
Apparently so, but it no longer works now. Dont know exactly why it was working before